operators/src/processing/reprojection.rs

```diff
@@ -657,6 +657,7 @@ where
                 tiling_strat.geo_transform,
                 self.tiling_spec.tile_size_in_pixels,
                 FillerTileCacheExpirationStrategy::DerivedFromSurroundingTiles,
+                FillerTimeBounds::from(query.time_interval),
```

Comment:
maybe add todo to derive the bounds from source stream?

Reply:
[1287d44](https://github.com/geo-engine/geoengine/pull/968/commits/1287d44f040444b0007238601f4f0fbf671a4e3f)